mxcubecore/model/lims_session.py

```diff
@@ -16,28 +16,43 @@
 
 
 class Proposal(BaseModel):
-    proposal_id: str = ""
-    person_id: str = ""
-    type: str = ""
-    code: str = ""
+    """Represents a proposal with key identifying details."""
+
+    proposal_id: str = ""  # Unique identifier for the proposal
+    person_id: str = ""  # Identifier of the person associated
+    type: str = ""  # Type/category of the proposal (e.g., "MX")
+    code: str = ""  # Specific code associated with the proposal e.g., "IH-LS"
+    # Proposal number; may be sequential or unique (uniqueness may depend on
+    # the combination of `code` and `number`)
```

Comment:
There are multiple ways to document class attributes.

This is probably the "best" one.
Maybe it is not the "best", but that is one that I know should work.

There are other ways we can explore if this one is not satisfying.

```suggestion
    """Proposal with key identifying details.
    
    Attributes:
        proposal_id: Unique identifier for the proposal.
        person_id: Identifier of the person associated.
        type: Type/category of the proposal (e.g., "MX").
        code: Specific code associated with the proposal e.g., "IH-LS".
        number: Proposal number; may be sequential or unique
            (uniqueness may depend on the combination of ``code`` and ``number``).
    """

    proposal_id: str = ""
    person_id: str = ""
    type: str = ""
    code: str = ""
    number: str = ""
```